test/data.jl

```diff
@@ -305,6 +305,24 @@ module TestData
     # @test (3 in unique(dv))
     # @test (NA in unique(dv))
 
+    # test nonunique() with extra argument
+    df1 = DataFrame(a = ["a", "b", "a", "b", "a", "b"], b = 1:6, c = [1:3;1:3])
+    df = vcat(df1, df1)
+    @test nonunique(df) == [rep(false,6); rep(true, 6)]
+    @test nonunique(df, :) == [rep(false,6); rep(true, 6)]
+    @test nonunique(df, Colon()) == [rep(false,6); rep(true, 6)]
+    @test nonunique(df, :a) == [rep(false,2); rep(true, 10)]
+    @test nonunique(df, [1, 3]) == [rep(false,6); rep(true, 6)]
+
+    # Test unique() with extra argument
+    @test unique(df) == df1
+    @test unique(df, :) == df1
+    @test unique(df, Colon()) == df1
+    @test unique(df, 2:3) == df1
+    @test unique(df, 3) == df1[1:3,:]
+    @test unique(df, [:a, :c]) == df1
```

Comment:
Same here: please add a test with a vector of integers, and with a single symbol.